packages/web/src/components/gcds-link/gcds-link.css

```diff
@@ -0,0 +1,43 @@
+:host a {
+  cursor: pointer;
+  color: var(--gcds-link-default);
+
+  background-color: var(--gcds-button-text-only-default-background);
+
+  &
+  :not([aria-disabled='true']) {
+    text-decoration: underline;
+    text-underline-offset: 0.2em;
+    text-decoration-color: currentColor;
+    text-decoration-thickness: var(--gcds-button-text-only-default-decoration-thickness);
+  }
+
+  &
+  :hover {
+    background-color: var(--gcds-button-text-only-default-background);
+    color: var(--gcds-link-hover);
+    text-decoration-thickness: var(--gcds-button-text-only-hover-decoration-thickness);
+  }
+
+  &
```

Comment:
For these selectors to work, you will need to format the CSS like `&:focus`.  Right now the component doesn't render the proper styles on hover and focus.

Reply:
Thanks, just noticed I think my editor is not configured properly for CSS; don't know why it reformatted my file to this!